ipatests/test_integration/test_user_permissions.py

```diff
@@ -68,6 +68,40 @@ def test_delete_preserve_as_alternate_admin(self):
         # call ipa user-del --preserve
         self.master.run_command(['ipa', 'user-del', '--preserve', testuser])
 
+    def test_selinux_user_optimized(self):
+        """
+        check if username in selinux are not optimized out
+        related ticket https://pagure.io/SSSD/sssd/issue/3819
+        """
+
+        testuser = 'testuser_selinux'
+        password = 'Secret123'
+        testuser_password_confirmation = "%s\n%s\n" % (password,
+                                                       password)
+        self.master.run_command(['ipa', 'user-add', testuser,
+                                 '--first', testuser,
+                                 '--last', testuser,
+                                 '--password',
+                                 '--homedir',
+                                 '/root/{}'.format(testuser)],
+                                stdin_text=testuser_password_confirmation)
+
+        self.master.run_command(
+            'sshpass -p %s '
```

Comment:
sshpass may not be installed, you should test first if the file /usr/bin/sshpass is present or skip the test. Please see `ipatests/test_integration/test_legacy_clients.py` for an example.